EIPS/eip-725.md

```diff
@@ -1,55 +1,93 @@
----
+```
 eip: 725
-title: Proxy Account
-author: Fabian Vogelsteller (@frozeman), Tyler Yasaka (@tyleryasaka)
+title: ERC-725 Smart Contract Based Account
+author: Fabian Vogelsteller <fabian@lukso.network>, Tyler Yasaka (@tyleryasaka)
 discussions-to: https://github.com/ethereum/EIPs/issues/725
 status: Draft
 type: Standards Track
 category: ERC
+requires: ERC165, ERC173, ERC1271 (optional)
```

Comment:
Please list ERCs and EIPs using only their number. E.g. `requires: 165, 173`. It isn't clear via EIP-1 how to denote optional EIPs, but considering it doesn't appear to be core to this EIP I recommend omitting it from the list of required EIPs.